lib/src/Uncertainty/Algorithm/MetaModel/LinearModel/openturns/LinearModelResult.hxx

```diff
@@ -103,6 +106,9 @@ public:
   /** Cook distance accessor */
   virtual Point getCookDistances() const;
 
+  /** Sigma2 accessor */
+  virtual Scalar getSigma2() const;
```

Comment:
should be name this getVariance ?

Reply:
The name of the attribute is sigma2, hence the name of the method. Would you agree on renaming the attribute too to keep this consistent?

Reply:
getResidualVariance

Reply:
Fixed at 15f769d81